packages/wct-mocha/src/environment/helpers.ts

```diff
@@ -25,7 +25,7 @@ const nativeRequestAnimationFrame = window.requestAnimationFrame;
  * @param {function()} callback
  * @param {function()} stepFn
  */
-function safeStep(callback: (error?: any) => void, stepFn: () => void) {
+export function safeStep(callback: (error?: any) => void, stepFn: () => void) {
```

Comment:
there's an `export {}` at the top of this file you can remove

Reply:
Thanks!